Framework/WorkflowAlgorithms/src/SANSSolidAngleCorrection.cpp

```diff
@@ -44,6 +46,16 @@ static double getYTubeAngle(const SpectrumInfo &spectrumInfo, size_t i) {
   return sampleDetVec.angle(inPlane);
 }
 
+} // namespace
+
+//----------------------------------------------------------------------------------------------
+/** Constructor
+ */
+SANSSolidAngleCorrection::SANSSolidAngleCorrection() {
+  useAlgorithm("SolidAngle");
+  deprecatedDate("2019-06-15");
```

Comment:
I don't think we can deprecate this now. This algorithm was written to fit into the current SANS reduction workflow (see how it takes a `ReductionProperties` property as an input). So we can't just drop in `SolidAngle` in its place. It should be deprecated when we are ready to move to our new implementation, which won't be ready for a few months.